lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs

```diff
@@ -203,6 +211,17 @@ public IMRUJobDefinitionBuilder SetUpdateTaskMemory(int memory)
         }
 
         /// <summary>
+        /// Sets Per Map Configuration
```

Comment:
Yes you are right. So do I need to change anything or this is fine.
On Aug 28, 2015 4:17 PM, "Markus Weimer" <notifications@github.com> wrote:

> In lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs
> <https://github.com/apache/incubator-reef/pull/417#discussion_r38253485>:
>
> > @@ -203,6 +211,17 @@ public IMRUJobDefinitionBuilder SetUpdateTaskMemory(int memory)
> >          }
> >
> >          /// <summary>
> > +        /// Sets Per Map Configuration
>
> I'm thinking the IMRUDriver could have a constructor parameter of type
> Set<IPerMapConfiguration> directly. Hence, the IConfiguration that
> instantiates those can be merged straight with the DriverConfiguration.
>
> I think the confusion is mine, because we don't currently expose any way
> for the users of IMRU to directly add things to the Driver configuration,
> right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/417/files#r38253485>.
>